x-pack/filebeat/docs/inputs/input-aws-s3.asciidoc

```diff
@@ -75,6 +75,28 @@ create a notification through SQS. Please see
 https://docs.aws.amazon.com/AmazonS3/latest/dev/ways-to-add-notification-config-to-bucket.html#step1-create-sqs-queue-for-notification[create-sqs-queue-for-notification]
 for more details.
 
+[float]
+=== Parallel Processing
+Multiple Filebeat instances can read from the same SQS queues at the same time. This
```

Comment:
We should be more direct here. I also wonder if all these details are required because they make the content harder to read. How about this instead?

<hr>
To horizontally scale processing when there are large amounts of log data
flowing into an S3 bucket, you can run multiple {beatname_uc} instances that
read from the same SQS queues at the same time. No additional configuration is
required.
<hr>

BTW, the words "amount" and "number" have different usages. Your original sentence would have required "number" which sounded a little off, so I edited it. Here's a grammar explanation if you're interested: https://www.enago.com/academy/understanding-the-difference-between-number-and-amount/

Reply:
Good to know, done. Thank you!